pkg/events/derive/symbols_loaded.go

```diff
@@ -16,13 +16,20 @@ func SymbolsLoaded(soLoader sharedobjs.DynamicSymbolsLoader, watchedSymbols []st
 
 // Most specific paths should be at the top, to prevent bugs with iterations over the list
 var knownLibrariesDirs = []string{
-	"/usr/lib/x86_64-linux-gnu/",
 	"/usr/lib64/",
 	"/usr/lib/",
 	"/lib64/",
 	"/lib/",
 }
 
+var knownArchitectureDirs = []string{
+	"", // non-specific architecture dir
```

Comment:
Would this usually be skipped or not? If so maybe it should be last as range will stop before?

Reply:
Usually the libs are in the general directory, not in the arch specific directory.
The arch directories are not relevant to all distributions from what I have read.
In my opinion (which I didn't test) it is probably more efficient to keep it in this order.

Reply:
I'm afraid it is the opposite nowadays. If you do a:

```
$ for lib in $(pkg-config --list-all | awk '{print $1}'); do find /usr/lib | grep $lib; done
```

you will see most of the libs placed in `/usr/lib/$(uname -m)-linux-gnu/`, no ? This is because most of the distros (big ones) want to be able to cross compile things and they split libs per arch.

Reply:
Interesting.
OK, I will change it so the empty string will be the last one to be checked.
Thanks @NDStrahilevitz @rafaeldtinoco